tools/cxbe/Main.cpp

```diff
@@ -19,19 +20,33 @@ int main(int argc, char *argv[])
     char szXbeFilename[OPTION_LEN + 1] = { 0 };
     char szDumpFilename[OPTION_LEN + 1] = { 0 };
     char szXbeTitle[OPTION_LEN + 1] = "Untitled";
+    char szXbeTitleID[OPTION_LEN + 1] = "";
+    char szXbeRegions[OPTION_LEN + 1] = "";
+    char szXbeVersion[OPTION_LEN + 1] = "";
     char szMode[OPTION_LEN + 1] = "retail";
     char szLogo[OPTION_LEN + 1] = "";
+
     bool bRetail;
+    uint32 dwTitleId = 0x4358270F; // CX-9999
+    uint32 dwRegions;
+    uint32 dwVersion;
 
     const char *program = argv[0];
     const char *program_desc = "CXBE EXE to XBE (win32 to Xbox) Relinker (Version: " VERSION ")";
-    Option options[] = { { szExeFilename, NULL, "exefile" },
-                         { szXbeFilename, "OUT", "filename" },
-                         { szDumpFilename, "DUMPINFO", "filename" },
-                         { szXbeTitle, "TITLE", "title" },
-                         { szMode, "MODE", "{debug|retail}" },
-                         { szLogo, "LOGO", "filename" },
-                         { NULL } };
+    Option options[] = {
+        { szExeFilename, NULL, "exefile" },
+        { szXbeFilename, "OUT", "filename" },
+        { szDumpFilename, "DUMPINFO", "filename" },
+        { szXbeTitle, "TITLE", "title" },
+        { szXbeTitleID, "TITLEID", "{%c%c-%u|%x}" },
```

Comment:
What does the `%` do here?

Reply:
Nothing
It's to let the user know what format to write the ID in (<char><char>-<uint> or <hex>)